src/main/java/student/management/StudentManagement/exception/NotFoundByIdException.java

```diff
@@ -0,0 +1,20 @@
+package student.management.StudentManagement.exception;
+
+public class NotFoundByIdException extends RuntimeException {

```

Comment:
用途をID検索に絞る必要はないと思いました。
```suggestion
public class NotFoundException extends RuntimeException {

```

Reply:
NotFoundExceptionに変更しました！

[9d436fe]

Reply:
9d436feb1bdc5c0b3c64cd35d92aa84ce683b7b6